Automattic/woocommerce-payments
# Implement payments via Express Checkout Element for Simple Products on the Product Page

Fixes #8868 

#### Changes proposed in this Pull Request

Implements payments using the Express Checkout Elements API.
This feature is behind the `_wcpay_feature_stripe_ece` flag.

#### Testing instructions

Note that the changes on this PR are the same as #8914. So, please  make sure to follow the setup instructions before starting to test this PR.

Testing on this PR must focus on the product page for **simple products** only.

> [!NOTE]
> There shouldn't be any problem with testing Google Pay or Apple Pay using your own credit card. **Just make sure the site is in test mode**.

Visit the product page for a simple product then:

**Pay using Google Pay**
- Verify shipping address can be changed.
- Verify the order is created with correct amount.
- Verify line items are present in the payment sheet.
- Verify tax is applied to the totals.
- Test with a US shipping address.
- Test with a US billing address.


**Pay using Apple Pay**
- Verify shipping address can be changed.
- Verify the order is created with correct amount.
- Verify line items are present in the payment sheet.
- Verify tax is applied to the totals.
- Test with a US shipping address.
- Test with a US billing address.

-------------------


**Post merge**
